README.md

```diff
@@ -88,7 +88,7 @@ dc.open(() => {
   const onReady = () => {
     const contactId = dc.createContact('homie', 'friend@site.org')
     const chatId = dc.createChatByContactId(contactId)
-    dc.sendTextMessage(chatId, 'Hi!')
+    dc.sendMessage(chatId, 'Hi!')
```

Comment:
This is also incorrect. But could be correct if we rewrote `sendMessage()` to check second parameter for string.

Reply:
E.g. something like this

```js
sendMessage (chatId, msg) {                                                
  if (!msg) {                                               
    throw new Error('invalid msg parameter')         
  }
  if (typeof msg === 'string') {
    msg = new Message(msg)
  } else if (!msg.dc_msg) {
    throw new Error('invalid msg object')         
  }                                                                        
  return binding.dcn_send_msg(this.dcn_context, Number(chatId), msg.dc_msg)
}                                                                          
```